backend/routes/follows.js

```diff
@@ -0,0 +1,48 @@
+import express from "express";
+import { Follow } from "../models/follow.js";
+import { User } from "../models/user.js";
+import { Stock } from "../models/stock.js";
+
+const router = express.Router();
+
+router.post("/follow", async (req, res) => {
+    const ticker = req.body.ticker.toUpperCase();
+
+    const user = req.session.user;
```

Comment:
Not exactly here, but I believe all routes (except login and signup) require an active session in order to work. 
Let's add a middleware in all routes that checks for session OR responds "missing session".